kpi/serializers/v2/asset.py

```diff
@@ -720,3 +745,86 @@ class AssetUrlListSerializer(AssetSerializer):
 
     class Meta(AssetSerializer.Meta):
         fields = ('url',)
+
+
+class AssetMetadataListSerializer(AssetSerializer):
+
+    date_latest_deployement = serializers.SerializerMethodField()
+    date_first_deployement = serializers.SerializerMethodField()
+    languages = serializers.SerializerMethodField()
+    owner__name = serializers.SerializerMethodField()
+    owner__email = serializers.SerializerMethodField()
+    owner__organization = serializers.SerializerMethodField()
+
+    class Meta(AssetSerializer.Meta):
+        fields = (
+            'url',
+            'date_modified',
+            'date_created',
+            'date_latest_deployement',
+            'date_first_deployement',
+            'owner',
+            'owner__username',
+            'owner__email',
+            'owner__name',
+            'owner__organization',
+            'uid',
+            'kind',
+            'name',
+            'settings',
+            'languages',
+            'asset_type',
+            'version_id',
+            'version_count',
+            'has_deployment',
+            'deployed_version_id',
+            'deployment__active',
+            'deployment__submission_count',
+            'permissions',
+            'status',
+            'data_sharing',
+            'data',
+        )
+
+    def get_data(self, *args: list, **kwargs: dict) -> str:
+        if view_has_perm(self._get_view(), PERM_VIEW_SUBMISSIONS):
+            return super().get_data(*args, **kwargs)
+        return ''
+
+    def get_date_first_deployement(self, obj: Asset) -> Optional[datetime.datetime]:
+        version = obj.asset_versions.filter(deployed=True).last()
+        if version:
+            return version.date_modified
+
+    def get_date_latest_deployement(self, obj: Asset) -> Optional[datetime.datetime]:
+        version = obj.asset_versions.filter(deployed=True).first()
+        if version:
+            return version.date_modified
+
+    def get_languages(self, obj: Asset) -> List[str]:
+        return obj.summary.get('languages', [])
+
+    def get_owner__email(self, obj: Asset) -> str:
+        return obj.owner.email
+
+    def get_owner__name(self, obj: Asset) -> str:
+        return self._get_user_detail(obj, 'name')
+
+    def get_owner__organization(self, obj: Asset) -> str:
+        return self._get_user_detail(obj, 'organization')
+
+    def get_permissions(self, *args: list, **kwargs: dict) -> list:
+        if view_has_perm(self._get_view(), 'view_permissions'):
```

Comment:
Should the user see the permissions of the asset if they are the owner even if the view does not that permission?

Reply:
Hmm good point — probably!